### DISCUSSION TITLE
music source feature

### DISCUSSION THREAD

**Discussion by mak333333** [HemantKArya/BloomeeTunes#77] [Ideas]
a music source feature similar to Spotube for the odd time a song refuses to play you can just choose an alternate source would be amazing

**Comment by HemantKArya** [comment]
yeah, that will be a good idea. to alternate the source within the song but before that I think we have to rethink how to find the exact song for the same song we are changing source, because I found in some apps it sometimes plays wrong music. But still if anyone have an idea, please share it here.

**Comment by BVISHNU78** [comment]
Please add select the album of song from playlist and also selecting singer other songs from any playlist it is missing.

**Reply by BVISHNU78** [reply]
Still fetching issue Presist and aslo error issue coming when playing songs

**Reply by HemantKArya** [reply]
Working right now? can you open an issue with screenshots.

**Reply by BVISHNU78** [reply]
![Screenshot_20250215-210726.jpg](https://github.com/user-attachments/assets/9805c62a-054c-495e-9472-c0a4d99ffbb9)

![Screenshot_20250215-210716.jpg](https://github.com/user-attachments/assets/42f4f5f4-1fc2-4d0a-a2be-282303455e4b)

**Reply by BVISHNU78** [reply]
https://developer.spotify.com/documentation                                    https://developer.amazon.com/docs/music/API_browse_overview.html is free music api like saavan

**Reply by BVISHNU78** [reply]
https://publicapis.io/gaana-api  and https://github.com/ZingyTomato/GaanaPy

**Comment by BVISHNU78** [comment]
![Screenshot_20250215-210447.jpg](https://github.com/user-attachments/assets/772012fb-f2aa-4734-84ee-518e543e8b1f)

No option to choose album only album view on original link

**Comment by BVISHNU78** [comment]
Please add jio saavan to play playlist instead of YouTube other wise automatically selection if YouTube cannot fetch the media. Other wise manually give an option to select
